contributors/devel/sig-architecture/api_changes.md

```diff
@@ -845,6 +845,13 @@ local-up-cluster.sh, kind, etc) and running `kubectl get
 <resource>.<version>.<group>`.
 * [Integration tests](../sig-testing/integration-tests.md)
 are also good for testing the full CRUD lifecycle along with the controller.
+  * To write integration tests for beta APIs you will need to selectively enable the resources you need.
+    You can do this using [test/integration/framework/controlplane_utils.go#NewIntegrationTestControlPlaneConfigWithOptions](https://github.com/kubernetes/kubernetes/blob/2b1b849d6a8bdeb7dc0807438cfd0ff2a9d752c1/test/integration/framework/controlplane_utils.go#L287).
```

Comment:
I don't feel strongly, but I've lost track of which integration invocation is newer / more favored, and wondered if we should point people at apitesting.StartTestServerOrDie instead ... it's nice because you can just pass in flags, like `--runtime-config=$mygroup/v1beta1=true`

Reply:
updated.